pkg/jx/cmd/controller_build.go

```diff
@@ -321,6 +328,116 @@ func (o *ControllerBuildOptions) createPromoteStepActivityKey(buildName string,
 	}
 }
 
+// completeBuildSourceInfo sets the PR author and PR title from GitHub in the given PA
+// If the PA is a branch build it then sets the commit author and last commit message
+func (o *ControllerBuildOptions) completeBuildSourceInfo(activity *v1.PipelineActivity) {
+
+	log.Infof("[BuildInfo] Completing build info for PipelineActivity=%s\n", activity.Name)
```

Comment:
nit: I would consider adding a filed fortmatter instead of adding the context in each log:
```
logrusutil.NewDefaultFieldsFormatter(nil, logrus.Fields{"component": "BuildInfo"})

// NewDefaultFieldsFormatter returns a DefaultFieldsFormatter,
// if wrappedFormatter is nil &logrus.JSONFormatter{} will be used instead
func NewDefaultFieldsFormatter(
	wrappedFormatter logrus.Formatter, defaultFields logrus.Fields,
) *DefaultFieldsFormatter {
	res := &DefaultFieldsFormatter{
		WrappedFormatter: wrappedFormatter,
		DefaultFields:    defaultFields,
	}
	if res.WrappedFormatter == nil {
		res.WrappedFormatter = &logrus.JSONFormatter{}
	}
	return res
}
```

Reply:
This is not using `logrus`, migrating the the whole project to use it should be another PR.